hack/test-e2e.sh

```diff
@@ -20,6 +20,8 @@ popd
 # update the manifest with the image built by ci
 sed -i "s,quay.io/openshift/origin-elasticsearch-operator:latest,${IMAGE_ELASTICSEARCH_OPERATOR}," ${manifest}
 
+sudo sysctl -w vm.max_map_count=262144 ||:
```

Comment:
need to skip this if `REMOTE_CLUSTER` is true

Reply:
updated to skip as necessary